### ISSUE TITLE
Some characters in the address linked from TOC are deleted and the link cannot be made.

### ISSUE THREAD

**Issue by TsNeko** [yzhang-gh/vscode-markdown#705]
#### What is the problem?
Some characters in TOC linking addresses have been removed so that it is no longer possible to link to a chapter from the TOC in the preview.

#### How can I reproduce it?

example of .md file
<pre>
&lt;!-- TOC depthFrom:1 depthTo:3 -->

- [１・２](#１２)
 
&lt;!-- /TOC -->

# １・２
</pre>

Presumably, `#１２` is bad and `#１・２` is right.
The unicode of ・ is U+30FB.

**Comment by yzhang-gh** [comment]
Actually on GitHub, the anchor is `#１２`. Are you using Markdown somewhere else?

![image](https://user-images.githubusercontent.com/7588612/83217823-30dec800-a19f-11ea-9d19-2707b8bae590.png)

BTW, it looks like you are also using another TOC extension. (This extension doesn't add/recognize  comments like `<!-- TOC ... -->`, `/TOC`.)

**Comment by TsNeko** [comment]
I solved this problem by matching the Markdown All in One setting to the Markdown preview feature built into the vscode.

![image01](https://user-images.githubusercontent.com/6040479/83231270-36033d80-a1c6-11ea-8c2e-e2b251caf97c.jpg)

Markdown All in One 3.0.0 recognizes &lt;!-- TOC ... -->.
I don't have Markdown TOC (AlanWalk) installed.

Also, &lt;!-- TOC ... --> in another .md file didn't recognized, so I installed Markdown TOC (AlanWalk).
To set slugifyMode to vscode works correctly.
